doc/nrf/templates/mesh_model_README.rst

```diff
@@ -0,0 +1,210 @@
+.. _bt_mesh_model_template:
+
+Mesh model template
+###################
+
+.. note::
+   Sections with * are optional.
+
+The XYZ Mesh models allow to remotely control a disco light with a mesh device.
+
+There are two XYZ models:
+
+    * XYZ Server
+    * XYZ Client
+
+.. tip::
+   Explain what this model allows you to do within the broader context of the mesh network.
+   Use max two sentences (full sentences, not sentence fragments).
+   You can go into more detail in the next section.
+   List server and client models that are part of this Mesh model.
```

Comment:
I liked your idea of adding a link to the common types here, if any. Perhaps we can work that in?